deploy/helm/listener-operator/templates/controller-deployment.yaml

```diff
@@ -34,7 +34,7 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- include "operator.resources" . | nindent 12 }}
```

Comment:
```suggestion
            {{- include "operator.controller.resources" . | nindent 12 }}
```

Reply:
This template is also not provided anywhere, did you forget to check in _helpers.tpl?

Reply:
That change will come through via templating once we merge https://github.com/stackabletech/operator-templating/pull/262